Fix TSNE page wording issues

lmcnulty opened the pull request:
Per #1020

cesarvarela commented on site/gatsby-site/src/components/cite/TsneVisualization.js:
```diff
@@ -385,7 +385,7 @@ const TsneVisualization = ({ currentIncidentId }) => {
           <div style={{ display: 'flex', gap: '1em', alignItems: 'center' }}>
             <label htmlFor="color-axis-select">
-              <Trans>Color by editor’s classification</Trans>
+              <Trans>Color by incident classifications from taxonomies</Trans>
```
Colores basados en la clasificación de los incidentes por su taxonomía

cesarvarela commented on site/gatsby-site/src/templates/tsneVisualizationPage.js:
```diff
@@ -29,13 +29,12 @@ function TsneVisulizationPage(props) {
         <Trans>
-          The visualization below plots incidents closer together according to the similarity of
-          their reports texts, as identified through{' '}
+          The visualization below shows incidents closer together when a{' '}
```
La siguiente visualización muestra los incidentes en proximidad proporcional a la similitud semántica de sus textos, identificados a través de <1>nuestro sistema de procesamiento de lenguaje natural</1>

cesarvarela reviewed (changes requested):
Does this need updates to the Spanish translations?

lmcnulty commented:
@cesarvarela I've added the changes to the translation. I also noticed that the title was in a `t()` but had no translation. I took a guess and added `"Spatial Visualization": "Visualización espacial"`, but let me know if that's not right.
